Makefile

```diff
@@ -183,6 +188,17 @@ endif
 	@gzcat $(GZ_FILE) | docker-compose run --rm web ./manage.py dbshell
 	@docker-compose run --rm web ./manage.py migrate --no-input
 
+fetchremotedb: $(shell if [ ! -d "./db_cache/" ]; then mkdir ./db_cache/; fi;)
```

Comment:
A more make-ish way to do this is:
```
fetchremotedb: db_cache

db_cache:
	mkdir -p db_cache
```

Reply:
Also given the number times "db_cache" appears, I would assign this to a make variable.  Say,
```
DB_CACHE_DIR = db_cache/
```
The you would do things like `${DB_CACHE_DIR}${DB_FILE_NAME}`

As a side note, I'm pretty sure the `./` isn't buying anything for you, but you can add it to the variable definition if you want.